src/compiler/checker.ts

```diff
@@ -5737,7 +5737,7 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
         return rightMeaning === SymbolFlags.Value ? SymbolFlags.Value : SymbolFlags.Namespace;
     }
 
-    function getAccessibleSymbolChain(symbol: Symbol | undefined, enclosingDeclaration: Node | undefined, meaning: SymbolFlags, useOnlyExternalAliasing: boolean, visitedSymbolTablesMap: Map<SymbolId, SymbolTable[]> = new Map()): Symbol[] | undefined {
+    function getAccessibleSymbolChain(symbol: Symbol | undefined, enclosingDeclaration: Node | undefined, meaning: SymbolFlags, useOnlyExternalAliasing: boolean, visitedSymbolTablesMap = new Map<SymbolId, SymbolTable[]>()): Symbol[] | undefined {

```

Comment:
I prefer the original style. Let me check with the team.

Reply:
Later: @jakebailey points out that these changes are a fraction of the sites that *could* change, which means the code base already mostly follows the rule, and these sites are the few that are inconsistent. So let's make the change.

Reply:
Yeah, the rule reports 338 errors when configured with https://typescript-eslint.io/rules/consistent-generic-constructors/#type-annotation.